CHANGELOG.md

```diff
@@ -6,21 +6,66 @@ accidentally triggering the load of a previous DB version.**
 
 ## 1.3.0 (unreleased)
 
+This release contains major new functionality that we call continuous aggregates.
+
+Aggregate queries which touch large swathes of time-series data can take a long
+time to compute because the system needs to scan large amounts of data on every
+query execution. Our continuous aggregates continuously calculate the
+results of a query in the background and materialize the results. Queries to the
+continuous aggregate view are then significantly faster as they do not need to
+touch the raw data in the hypertable, instead using the pre-computed aggregates
+in the view.
+
+Continuous aggregates are somewhat similar to PostgreSQL materialized
+views, but unlike a materialized view, continuous
+aggregates do not need to be refreshed manually; the view will be refreshed
+automatically in the background as new data is added, or old data is
+modified. Additionally, it does not need to re-calculate all of the data on
+every refresh. Only new and/or invalidated data will be calculated.  Since this
+re-aggregation is automatic, it doesn’t add any maintenance burden to your
+database.
+
+Another alternative approach would be to use triggers to update a materialized view
+immediately upon writing to the database, however this causes significant write
+amplification, which slows inserts. We have designed continuous aggregations to
```

Comment:
I don't think we need to address the trigger approach here.

Reply:
yes,  agree should leave out the trigger  part.

Reply:
I actually think the trigger part is useful because a lot of people use custom triggers for this sort of stuff and this would give them context.

Reply:
see now?